packages/core/src/feature-app-manager.ts

```diff
@@ -63,7 +75,8 @@ export interface FeatureAppManagerLike {
 
   getFeatureAppScope<TFeatureApp>(
     featureAppDefinition: FeatureAppDefinition<TFeatureApp>,
-    idSpecifier?: string
+    idSpecifier?: string,
```

Comment:
Maybe refactor it to use an options object. This scales better and is nicer to use.